README.org

```diff
@@ -61,6 +63,19 @@
      $ mv target/debug/liblspce_module.dylib lspce-module.dylib
    #+END_SRC
 
+** Installing using Straight
+   #+BEGIN_SRC elisp
+   (straight-use-package
+    `(lspce :type git :host github :repo "zbelial/lspce"
+            :files (:defaults ,(pcase system-type
+                                 (`gnu/linux "lspce-module.so")
+                                 (_ "lspce-module.dylib")))
+            :pre-build ("cargo" "build" "--release")
+            :post-build ,(pcase system-type
+                           (`gnu/linux '("cp" "./target/release/liblspce_module.so" "./lspce-module.so"))
+                           (_ '("cp" "./target/release/liblspce_module.dylib" "./lspce-module.dylib")))))
```

Comment:
As a result of `cargo build` `liblspce_module.d` is produced on both macOS and Linux. I am not familiar with rust, but it seems that lspce works well without `liblspce_module.d`.  
Thus I am not sure that `liblspce_module.d`. should be installed anywhere.

Reply:
Thanks. 
The "_" in the pcase means any platforms except gnu/linux, right? (I'm not familiar with pcase) If so, I think we can specify macos here and leave windows alone until some people use straight on windows to improve it. 

I use lspce on Linux and test it on Windows sometimes, on both of them lspce works well without `lspce_module.d`, so I think there's no need to install it somewhere.

Reply:
Done.  

Also:
- fixed subtle issue happened during clean reinstall of the package (as it's turned out we have to run cp in pre-build state because lisp code requires lspce-module before compilation).
- Corrected style (` vs ') and indent.